docs/architecture/decisions/0004-use-aws-load-balancer-controller-for-edge-traffic-services.md

```diff
@@ -0,0 +1,35 @@
+# 4. Use AWS Load Balancer Controller for edge traffic services
+
+Date: 2021-08-12
+
+## Status
+
+Accepted
+
+## Context
+
+We require a method of managing and direcing external internet traffic into the cluster. Kubernetes provides [several options for handling inbound traffic](https://medium.com/google-cloud/kubernetes-nodeport-vs-loadbalancer-vs-ingress-when-should-i-use-what-922f010849e0).
```

Comment:
Nice that we got the same reference here when we were trying to explore the different options for getting external traffic inside the cluster.